File-Formats/DocIO/Working-with-Charts/Format-Legend.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Modify the appearance of legend | Syncfusion
+description: Learn how to modify the appearance of legend in a chart in a  Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Legend
+
+Legends are visual **pictorial** hints that provide a viewer information that helps them understand an chart. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library you can customize the **legend** in the chart.
+
+## Set the Position of Legend
+
+The following code snippet illustrates how to set the legend position in chart.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Set the position of legend.
+chart.Legend.Position = OfficeLegendPosition.Bottom;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the layout Inclusion
+
+The following code snippet illustrates how to prevent the overlapping the legend in chart.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Legend without overlapping the chart.
+ chart.Legend.IncludeInLayout = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Border
+
+The following code snippet illustrates how to modify the border of the legend in chart.
```

Comment:
The following code sample illustrates how to modify the border of the legend in chart.